src/index.html

```diff
@@ -5,7 +5,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Leaderboard</title>
 </head>
-<body>
+<body id="body">
 	<header class="header">
 		<h1 class="title">Leaderboard</h1>
 	</header>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add the following features on your form elements:
```HTML
             <form class="form">
	        <input type="text" name="name" id="name" placeholder="Enter Your Name">
	        <input type="text" name="score" id="score" placeholder="Enter Your Score">
	        <button type="submit" id="submit">Submit</button>
	     </form>
```
- A feature to clear all the `inputs` once you added a new score
- Update the `type="text"` to `type="number"` of the `score input`.